laravel/framework/2018-03-30-1.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Exploit of encryption failure vulnerability
+link:      https://medium.com/@taylorotwell/laravel-security-release-5-6-15-and-5-5-40-56f1257933a0
+cve:       ~
+branches:
+    5.5.x:
+        time:     2018-03-30 14:15:00
+        versions: ['>=5.5.0', '<5.5.40']
+    5.6.x:
+        time:     2018-03-30 14:15:00
```

Comment:
It seems to me that the commit fixing the issue is laravel/framework@886d261df0854426b4662b7ed5db6a1c575a4279 which was made at 13:24.

Reply:
Thanks again, [see this comment](https://github.com/FriendsOfPHP/security-advisories/pull/279/files/0033f23237a63b3bcb13c51955b047a0e0a7f35f#r178441281).